### PR TITLE
Fix unity3d.com #222

### PR DESCRIPTION
**Pull request by ghost**
- Add backslash to ubuntu.com url
- Add company Unity Technologies
- Link Unity Ads and Unity Technologies under the same companyId
- Fix Unity Ads websiteUrl
- Add Unity domains

### CODE REVIEW

File: source/companies.json [hunk 1]
```diff
@@ -228,9 +228,19 @@
...
+            "websiteUrl": "https://ubuntu.com/",
             "description": "Ubuntu is an ancient African word meaning ‘humanity to others’. It is often described as reminding us that ‘I am what I am because of who we all are’. We bring the spirit of Ubuntu to the world of computers and software. The Ubuntu distribution represents the best of what the world’s software community has shared with the world."
-        },                
+        },     
+        "unity": {
+            "name": "Unity Technologies",
+            "url": "https://unity.com/",
+            "description": "Unity is a cross-platform game engine developed by Unity Technologies, first announced and released in June 2005 at Apple Worldwide Developers Conference as a Mac OS X game engine. The engine has since been gradually extended to support a variety of desktop, mobile, console and virtual reality platforms. It is particularly popular for iOS and Android mobile game development, is considered easy to use for beginner developers, and is popular for indie game development."
+        },           
+        "unity_ads": {
```
Comment (jellizaveta) [inline source/companies.json:239]: `companies.json` contains a list of companies, so in my opinion it is enough to add only `Unity Technologies` and specify `Unity Ads` in the trackers

**Review by jellizaveta — CHANGES_REQUESTED** [review]

File: source/trackers.json [hunk 2]
```diff
@@ -509,8 +509,8 @@
         "unity_ads": {
             "name": "Unity Ads",
             "categoryId": 4,
-            "url": "https://unity.com/solutions/mobile-business/monetize-your-game",
-            "companyId": null
+            "url": "https://unity.com/products/unity-ads/",
+            "companyId": "unity"
         },
```
Comment (jellizaveta) [inline source/trackers.json:514]: just unity can be added in trackers:
```json
"unity": {
    "name": "Unity",
    "categoryId": 8,
    "url": "https://unity.com/",
    "companyId": "unity"
},
```

File: source/trackers.json [hunk 3]
```diff
@@ -889,6 +889,9 @@
         "snapcraftcontent.com": "ubuntu",
         "ubuntu.com": "ubuntu",
         "ubuntucompanyservices.co.za": "ubuntu",
+        "unity.com": "unity_ads",
```
Comment (jellizaveta) [inline source/trackers.json:892]:
```json
 "unity.com": "unity",
 ```

**Review by jellizaveta — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

File: source/trackers.json [hunk 4]
```diff
@@ -530,11 +530,11 @@
             "url": "https://ubuntu.com/",
             "companyId": "ubuntu"
         },
-        "unity_ads": {
-            "name": "Unity Ads",
-            "categoryId": 4,
-            "url": "https://unity.com/solutions/mobile-business/monetize-your-game",
-            "companyId": null
```
Comment (jellizaveta) [inline source/trackers.json:537]:
```json
"unity_ads": {
    "name": "Unity Ads",
    "categoryId": 4,
    "url": "https://unity.com/solutions/mobile-business/monetize-your-game",
    "companyId": "unity"
},
"unity": {
    "name": "Unity",
    "categoryId": 8,
    "url": "https://unity.com/",
    "companyId": "unity"
},
```

File: source/trackers.json [hunk 5]
```diff
@@ -937,7 +937,10 @@
         "snapcraftcontent.com": "ubuntu",
         "ubuntu.com": "ubuntu",
         "ubuntucompanyservices.co.za": "ubuntu",
-        "unityads.unity3d.com": "unity_ads",
```
Comment (jellizaveta) [inline source/trackers.json:940]: this is correct, as long as the `unity_ads` tracker exists
`"unityads.unity3d.com": "unity_ads",`

**Review by jellizaveta — CHANGES_REQUESTED** [review]

**Review by jellizaveta — APPROVED** [review]

**Pull request merged**